src/backend/Dockerfile

```diff
@@ -8,26 +8,20 @@ ENV PYTHONUNBUFFERED=1
 
 WORKDIR /app
 
-# Copy the backend source code, excluding the pretrained models (see .dockerignore)
-COPY . .
-
-RUN git submodule init && git submodule update --remote --merge
-
-# Download and unzip pretrained models
-RUN curl -L -o /app/src/backend/sinr/pretrained_models.zip https://data.caltech.edu/records/dk5g7-rhq64/files/pretrained_models.zip?download=1 && \
-    unzip -o /app/src/backend/sinr/pretrained_models.zip -d /app/src/backend/sinr && \
-    rm /app/src/backend/sinr/pretrained_models.zip
+# Download pretrained models
+RUN curl -L -o /app/pretrained_models.zip https://data.caltech.edu/records/dk5g7-rhq64/files/pretrained_models.zip?download=1
 
-# Create a new conda environment with Python 3.9
-RUN conda create -n inatator python=3.9
+COPY ./src/backend/requirements.txt .
 
 # Install Python dependencies
-RUN conda run -n inatator \
-    pip install -r ./src/backend/requirements.txt
+RUN pip install -r ./requirements.txt
 
-# Make the entrypoint script executable
-RUN chmod +x /app/src/backend/entrypoint.sh
+# Copy the backend source code, excluding the pretrained models (see .dockerignore)
+COPY . .
+
+RUN git submodule init && git submodule update --remote --merge
 
-EXPOSE 8000
```

Comment:
Is removing the `EXPOSE port` commands intentional? I noticed you did that in both Dockerfiles.

Reply:
I removed them because the ports are opened in the docker compos file, which seems to duplicate the functionality of this EXPOSE command. 
It seems to me better to keep the ports in docker images closed and manage them through one docker compos file?